docs-md/README.md

```diff
@@ -20,6 +20,7 @@
   * [Cordova/Ionic Native Plugins](cordova/using-cordova-plugins.md)
   * [Known Incompatible Plugins](cordova/known-incompatible-plugins.md)
 * Guides
+  * [Live Reload](guides/live-reload.md)
```

Comment:
Put it under Guides but would it be better under Basics? Live Reload is so useful....

Reply:
Guides is perfect. Live Reload isn't basic IMO